src/app/hooks/usePan.ts

```diff
@@ -15,12 +15,13 @@ export const usePan = (active: boolean) => {
   const [cursor, setCursor] = useState<'grab' | 'grabbing' | 'initial'>(
     active ? 'grab' : 'initial'
   );
+  const [status, setStatus] = useState<'idle' | 'moving'>('idle');
```

Comment:
I found that this is not needed as cursor `grabbing` state describe status `moving` and cursor other states describe status `idle` .